compiler/rustc_codegen_gcc/Readme.md

```diff
@@ -181,7 +181,7 @@ debug_tree(expr);
 
 (defined in print-tree.h)
 
-To print a debug reprensentation of a gimple struct:
+To print a debug representation of a gimple struct:
```

Comment:
i assume the maintainers of `rustc_codegen_gcc` probably want you to open this agains their [own repo](https://github.com/rust-lang/rustc_codegen_gcc). that's typically how nested repos in this one work.

Reply:
Yeah, I will revert and go to commit to the upstream.  Thanks for your suggestion. @compiler-errors